UI/window-basic-main-outputs.cpp

```diff
@@ -286,6 +286,9 @@ struct SimpleOutput : BasicOutputHandler {
 	void UpdateRecordingSettings_qsv11(int crf);
 	void UpdateRecordingSettings_nvenc(int cqp);
 	void UpdateRecordingSettings_amd_cqp(int cqp);
+#ifdef ENABLE_HEVC
+	void UpdateRecordingSettings_nvenc_hevc(int cqp);
+#endif
```

Comment:
Move this up to be next to the other nvenc function.

Reply:
Done.